addons/point_of_sale/static/src/js/Screens/TicketScreen/TicketScreen.js

```diff
@@ -648,6 +648,11 @@ odoo.define('point_of_sale.TicketScreen', function (require) {
                     args: [idsNotInCache],
                     context: this.env.session.user_context,
                 });
+                // Remove not loaded Order IDs
+                const fetchedOrderIdsSet = new Set(fetchedOrders.map(order => order.id));
+                const notLoadedIds = idsNotInCache.filter(id => !fetchedOrderIdsSet.has(id));
+                ids = ids.filter(id => !notLoadedIds.includes(id));
+
```

Comment:
It's not really a big problem but should we also adapt the totalCount ? Something like decrease the totalCount by the number of ids we do not fetch ? We should also check if doing this would cause a problem when there are a lot of paid orders and we have multiple pages

Reply:
I modified the code. But it loads all of the order in one request, no?

Reply:
@adgu-odoo I modified the code.

Reply:
Ok that, seems good for the total count. The thing is that I'm not sure it loads all the paid orders in one time as we give an offset and a limit when calling the `search_paid_order_ids`. I think the limit is fixed but the offset is computed on the page count and the page count is dependent on the total count. As this total count is not the same in the frontend than in the db (depending on user rights the frontend might show less orders they are in the db), maybe we could have overlaps (but the easiest way to know this is to test it I think we will be fixed directly). If you have question tell me on discord

Reply:
@adgu-odoo Upon testing, I discovered that altering the totalCount should be avoided. While this may result in fewer items displayed in the list, it allows for page navigation. Modifying the totalCount could lead to missing pages.